File: 백준/Bronze/2439. 별 찍기 － 2/별 찍기 － 2.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int count = sc.nextInt();

        for(int i = count-1 ; i >= 0; i--){
            for(int j =i; j >0 ; j--){
                System.out.printf(" ");
            }
            for(int j=count; j >i ; j--){
                System.out.printf("*");
            }
            System.out.println();
        }
    }
}

File: 백준/Bronze/2439. 별 찍기 － 2/별 찍기 － 2.cc
#include <iostream>

int main() {
    int count;
    std::cin >> count;

    for (int i = 1; i <= count; i++) {
        for (int j = 1; j <= count - i; ++j) {
            std::cout << " ";
        }


        for (int j = 1; j <= i; ++j) { ;
            std::cout << "*";
        }
        std::cout << std::endl;
    }
    return 0;
}